src/apps/demo-app/src/app/layout/layout-gap/layout-gap.component.ts

```diff
@@ -11,7 +11,7 @@ const DIRECTIONS = ['row', 'row-reverse', 'column', 'column-reverse'];
       </mat-card-subtitle>
       <mat-card-content class="large">
         <div fxFlexFill>
-          <div fxFlex
+          <div fxFlexFill
```

Comment:
Let's not include this change in this PR. This mixes fixes.

Reply:
Fixed